app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -163,39 +172,41 @@ Any financial support, for example a reduction in your fees to the rate paid by
 
-You can search by location, teaching subject and age group. You can also search for courses which pay a salary, and courses where student or skilled worker visa sponsorship is available.
+#### First apply for your training place, then apply for your visa
 
 If you need a visa to train to teach in England, you must have a confirmed offer of a training place on a course offering student or skilled worker visa sponsorship before you apply for your visa.
 
-Due to rules about immigration, not all providers can accept international applicants – contact them before you apply to check.
-
-#### Ways to train
-
-You can train in a school or in a university setting, and study full-time (over a year) or part-time (over 2 or more years). [Get a dedicated adviser](https://adviser-getintoteaching.education.gov.uk/) about the different types of courses available, or explore [how to train to be a teacher in England](/train-to-be-a-teacher).
+On [Find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses), you can search for courses where student or skilled worker visa sponsorship is available. You can also search by location, teaching subject and age group, and for courses which pay a salary.
 
-You may also wish to read [detailed guidance about fee-paying and salaried courses and qualified teacher status](https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#train).
+Due to rules about immigration, not all providers can accept international applicants – we suggest you contact them before you apply to check.
 
 <div class="inset">
   <h4>Already living in the UK?</h4>
 
   <p>
-If you have settled or pre-settled status under the <a href="https://www.gov.uk/settled-status-eu-citizens-families">EU Settlement Scheme</a>, or <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain</a>, you can apply to any teacher training provider in England.
+If you have settled or pre-settled status under the <a href="https://www.gov.uk/settled-status-eu-citizens-families">EU Settlement Scheme</a>, or <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain</a>, you can apply to any English teacher training provider.
   </p>
 </div>
 
-#### Your suitability for teaching
+#### Choosing a subject to teach and preparing to apply
+
+The subject you apply to teach must usually be the same, or closely related to, the subject you studied at university. (For example, engineering graduates can apply to train to teach physics, but computer science graduates are unlikely to be accepted to train to teach geography.) 
 
-As part of the application process, your training provider will confirm you’re suitable for teaching. They'll check:
+Your [teacher training adviser](#get-support-and-advice) can help with: 
```

Comment:
@laurahermionetennant92 I think this is using a jump link to a heading you've now removed (which is why it isn't building). Do you want to change the link or remove it?

Reply:
does it need to be changed to (#1-get-support-and-advice)?

Reply:
This seems to be working OK now :)